python/src/connection.rs

```diff
@@ -33,6 +34,65 @@ impl Connection {
             inner.table_names().await.infer_error()
         })
     }
+
+    pub fn create_table<'a>(
+        self_: PyRef<'a, Self>,
+        name: String,
+        mode: &str,
+        data: &PyAny,
+    ) -> PyResult<&'a PyAny> {
+        let inner = self_.inner.clone();
+
+        let mode = if mode == "create" {
+            CreateTableMode::Create
+        } else if mode == "overwrite" {
+            CreateTableMode::Overwrite
+        } else {
+            // TODO: allow users to pass in open options
+            CreateTableMode::exist_ok(|builder| builder)
+        };
```

Comment:
nit: seems like a good use for `match`?
```suggestion
        match mode {
            "create" => CreateTableMode::Create,
            "overwrite" => CreateTableMode::Overwrite,
            _ => CreateTableMode::exist_ok(|builder| builder)
        }
```

Reply:
Switched to match and put in a helper function to share with `create_empty_table`.  I also updated the node version to behave more similarly to the python version.